### ISSUE TITLE
PersesAtomMapper may broken with openff-tk 0.11

### ISSUE THREAD

**Issue by IAlibay** [OpenFreeEnergy/openfe#223]
As discussed in today's F@H call, see: https://github.com/choderalab/perses/issues/1127

**Comment by richardjgowers** [comment]
If so, it probably shows that our tests aren't adequate here, since I think we're onto ff tk 0.11

**Comment by IAlibay** [comment]
Yeah I think we went with the assumption that upstream works and we should just be able to import.

**Comment by atravitz** [comment]
this is outdated, closing.
